### PR TITLE
Run examples on CI

### PR DESCRIPTION
**Pull request by segiddins**

### CODE REVIEW

File: examples/meta-test.tr [hunk 1]
```diff
@@ -1,7 +1,7 @@
 type Foo {
     let bar: Int
     let baz: *Foo
-    let tup: (uint16_t, Bool)
+    let tup: (UInt16, Bool)
```
Comment (segiddins) [inline examples/meta-test.tr:4]: coercion doesn't work when the target type is `uint16_t`

File: examples/operator.tr [hunk 2]
```diff
@@ -6,9 +6,9 @@ func -(lhs: Int, rhs: Bool) -> Int {
   return lhs - (rhs ? 1 : 0)
 }
 
-func +(lhs: String, rhs: String) -> String {
+func +(lhs: String, rhs: Bool) -> String {
```
Comment (segiddins) [inline examples/operator.tr:9]: this was a redefinition

File: examples/meta-test.tr (hunk 1 above)
Comment (harlanhaskins) [reply examples/meta-test.tr:4]: Pretty sure we don't import `uint16_t` anymore...

File: examples/operator.tr (hunk 2 above)
Comment (harlanhaskins) [reply examples/operator.tr:9]: Oops. 👍

**Review by harlanhaskins — APPROVED** [review]
Overall it's excellent and I'm glad it's being run!

In the future, we probably want a Real Way™ to run tests and give them input automatically without adding them to the `run-examples.swift` file.

**Comment by segiddins** [comment]
> In the future, we probably want a Real Way™ to run tests and give them input automatically without adding them to the run-examples.swift file.

definitely! I just wanted something we can use to quickly sanity-check changes. and maybe so CI can be useful?

File: examples/meta-test.tr (hunk 1 above)
Comment (segiddins) [reply examples/meta-test.tr:4]: or that :P

**Comment by segiddins** [comment]
Do we have to explicitly link libdispatch or something? I have no idea why this is working locally but failing on CI

**Comment by segiddins** [comment]
This is now passing on CI!

**Comment by harlanhaskins** [comment]
Then are we good to merge????

**Comment by segiddins** [comment]
Yup!

**Pull request merged**